runtime/drivers/gcs/object_store.go

```diff
@@ -97,12 +97,8 @@ func (c *Connection) DownloadFiles(ctx context.Context, props map[string]any) (d
 			return nil, drivers.NewPermissionDeniedError(fmt.Sprintf("can't access remote err: %v", retrieveError))
 		}
 
-		anonIt, anonErr := rillblob.NewIterator(ctx, bucketObj, opts, c.logger)
-		if anonErr == nil {
-			iter = anonIt
-			err = nil
-		}
```

Comment:
This was added explicitly by this [PR](https://github.com/rilldata/rill/pull/5598) to retrieve from public bucket so do we revert it, does creds work for gcs public buckets ?

Reply:
Yeah but it looks like it was added by mistake, if you look at the surrounding code, it doesn't change any of the parameters, it just repeats the previous authenticated request.

Reply:
I'm pretty sure GCS doesn't require separate auth for public buckets, I believe they can be accessed normally with your normal credentials.

Reply:
Yes it doesn't look like its creating some anon request